project/Deps.scala

```diff
@@ -2,9 +2,9 @@ import sbt._
 
 object Deps {
 
-  val curatorFramework = "org.apache.curator" % "curator-framework" % "4.1.0"
-  val curatorClient = "org.apache.curator" % "curator-client" % "4.1.0"
-  val curatorDiscovery = "org.apache.curator" % "curator-x-discovery" % "4.1.0"
+  val curatorFramework = "org.apache.curator" % "curator-framework" % "5.2.0"
+  val curatorClient = "org.apache.curator" % "curator-client" % "5.2.0"
+  val curatorDiscovery = "org.apache.curator" % "curator-x-discovery" % "5.2.0"
```

Comment:
do we need to bump deps as part of this change?  I thought we were just adding a nozk configuration.

Reply:
I bumped this dep in order to get a version of Curator that doesn't rely on Zookeeper as a downstream dependency. It's easy enough to roll this back, and I left it because all the tests passed.

Reply:
what about the netty and boringssl bumps below?  is bumping those versions necessary?  I'd prefer to change as little as possible.

Reply:
@adleong I've rolled back the dependency versions, so please have a look and let me know if there are additional changes required